packages/workspace/src/generators/ci-workflow/__snapshots__/ci-workflow.spec.ts.snap

```diff
@@ -21,63 +20,24 @@ variables:
   HEAD_SHA: $(git rev-parse HEAD)
 
 jobs:
-  - job: agents
-    strategy:
-      parallel: 3
-    displayName: Nx Cloud Agent
-    pool:
-      vmImage: 'ubuntu-latest'
-    steps:
-      - script: npm install --prefix=$HOME/.local -g pnpm@8.2.0
-        displayName: Install PNPM
-      - script: pnpm install --frozen-lockfile
-        displayName: NPM Install Dependencies
-      - script: npx nx-cloud start-agent
-        displayName: Start Nx-Cloud agent
-
   - job: main
-    displayName: Nx Cloud Main
     pool:
       vmImage: 'ubuntu-latest'
     steps:
-      - script: npm install --prefix=$HOME/.local -g pnpm@8.2.0
-        displayName: Install PNPM
-      - script: pnpm install --frozen-lockfile
-        displayName: NPM Install Dependencies
-      - script: pnpm exec nx-cloud start-ci-run --stop-agents-after="build" --agent-count=3
-        displayName: Start CI run
-      - script: pnpm exec nx-cloud record -- pnpm exec nx format:check --base=$(BASE_SHA) --head=$(HEAD_SHA)
-        displayName: Check format
-      - script: pnpm exec nx affected --base=$(BASE_SHA) --head=$(HEAD_SHA) --target=lint --parallel=3
-        displayName: Run lint
-      - script: pnpm exec nx affected --base=$(BASE_SHA) --head=$(HEAD_SHA) --target=test --parallel=3 --ci --code-coverage
-        displayName: Run test
-      - script: pnpm exec nx affected --base=$(BASE_SHA) --head=$(HEAD_SHA) --target=build --parallel=3
-        displayName: Run build
-      - script: pnpm exec nx-cloud stop-all-agents
-        condition: always()
-        displayName: Stop all Nx-Cloud agents
+      - script: npm ci
+      # uncomment to enable task distribution
+      # - script: npx nx-cloud start-ci-run --distributes-on="5 linux-medium"
+      - script: npx nx-cloud record -- nx format:check --base=$(BASE_SHA) --head=$(HEAD_SHA)
+      - script: npx nx affected --base=$(BASE_SHA) --head=$(HEAD_SHA) --target=lint,test,build --parallel=3
```

Comment:
Let’s go with the short form -t

Victor prefers it as it’s visually cleaner but also makes sense not to emphasize target based on our recent discussion